ova/build.sh

```diff
@@ -41,11 +44,20 @@ step=$1; shift
 [ ! "$step" == "ova-ci" ] || [ ! "$step" == "ova-dev" ] || usage
 
 echo "--------------------------------------------------"
+if [ ! -d ${ROOT_UI_DIR} ]; then
+  export LATEST=$(curl -s https://api.github.com/repos/dispatchframework/dispatch-ui/releases/latest | jq -r .name)
```

Comment:
add `echo` command saying "UI directory does not exist, downloading UI release..."